src/Altinn.Notifications.Core/Models/SendCondition/ConditionClientError.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Altinn.Notifications.Core.Models.SendCondition
+{
+    /// <summary>
+    /// Model describing a condition client error
+    /// </summary>
+    public class ConditionClientError
+    {
+        /// <summary>
+        /// The result code of the condition check request
+        /// </summary>
+        public int? StatusCode { get; set; }
```

Comment:
Do we need these two properties to be nullable?

Reply:
My thought was that status code doesn't always make sense to add.. if response is 200 ok for instance. Or if we get a timeout and there never is a status code. And a message might not make sense to add if we get a 403 for instance, that's why I made them both nullable for now.

Reply:
Yeah, okay. I can live with that.

Ideally I don't think Core should care about https status codes, but I'm not going to spend the time on defining an abstraction and mapping.